ckanext/canada/public/css/canada1.css

```diff
@@ -359,4 +359,24 @@ a#edit-comment-body-0-format-help-about, button#edit-submit,button#edit-actions-
 
     color: white;
 
+}
+
+.user-image-wrapper > .image { 
+    position: relative;
+    padding-top: calc(100% + 10px);
+}
+@media all and (max-width: 992px){
+    .user-image-wrapper > .image { 
+        position: relative;
+        padding-top: 250px;
+    }
+}
+.user-image-wrapper img.user-image {
+    position: absolute;
+    max-height: 100%;
+    max-width: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
would you add some comments to indicate what these are intended to fix?

Reply:
@wardi this had to do with the user avatars. It has been removed now.